lib/Doctrine/ORM/QueryBuilder.php

```diff
@@ -797,13 +794,11 @@ public function addSelect($select = null)
     {
         $this->_type = self::SELECT;
 
-        if (empty($select)) {
+        if ($select === []) {
```

Comment:
Was this method supposed to be variadic too that you changed this check?

Reply:
I don't really understand your question, but I think it was supposed to be variadic too since there was a call to `func_get_args()`

Reply:
Oh… right!

Reply:
ohlol, nice catch

Reply:
I've checked the rest has updated signature and docblock